main/SQLHelper.h

```diff
@@ -321,21 +321,23 @@ typedef std::vector<std::string> TSqlRowQuery;
 // result for an sql query : Vector of TSqlRowQuery
 typedef std::vector<TSqlRowQuery> TSqlQueryResult;
 
+typedef   std::map<std::string, std::string> TOptionMap;
+
```

Comment:
You should use
```suggestion
using TOptionMap = std::map<std::string, std::string>;
```
in new code https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rt-using

Reply:
ok done